wbia/__init__.py

```diff
@@ -374,17 +374,6 @@ def reload_subs(verbose=True):
 rrrr = reload_subs
 
 
-from wbia.control.DB_SCHEMA_CURRENT import VERSION_CURRENT
```

Comment:
I want to revert this change.  The version of the database needs to be disassociated with the version of the WBIA codebase.